packages/grafana-data/src/types/logs.ts

```diff
@@ -36,6 +36,7 @@ export interface LogsMetaItem {
 }
 
 export interface LogRowModel {
+  dataFrameRow: any;
```

Comment:
This is needed to propagate all the data from the row downstream. This is needed for both the derived fields extractors and also for https://github.com/grafana/grafana/pull/20034 to work properly. This though needs some better thought as this is very provisional right now.

cc @torkelo @ivanahuckova